docs/pages/resources.rst

```diff
@@ -0,0 +1,131 @@
+.. _resources:
+
+Resources
+=========
+
+.. automodule:: runeq.resources
+
+For example usage patterns, see :ref:`quickstart`.
+
+API Clients
+-----------
+
+.. automodule:: runeq.resources.client
+
+.. autofunction:: runeq.resources.client.initialize
+.. autofunction:: runeq.resources.client.global_graph_client
+.. autofunction:: runeq.resources.client.global_stream_client
+
+.. autoclass:: runeq.resources.client.GraphClient
+   :special-members: __init__
+   :members:
+
+.. autoclass:: runeq.resources.client.StreamClient
+   :special-members: __init__
+   :members:
+
+Patient Metadata
+----------------
+
+.. automodule:: runeq.resources.patient
+
+Patients
+********
+
+.. autofunction:: get_patient
+.. autofunction:: get_all_patients
+
+.. autoclass:: Patient
+   :special-members: __init__
+   :members:
+   :inherited-members:
+.. autoclass:: PatientSet
+   :special-members: __init__
+   :members:
+   :inherited-members:
+
+
+Devices
+*******
+
+.. autofunction:: get_device
+.. autofunction:: get_all_devices
+
+.. autoclass:: Device
+   :special-members: __init__
+   :members:
+   :inherited-members:
+.. autoclass:: DeviceSet
+   :special-members: __init__
+   :members:
+   :inherited-members:
+
+Organization Metadata
+---------------------
+
+.. automodule:: runeq.resources.org
+
+.. autofunction:: get_org
+.. autofunction:: get_orgs
+
+.. autoclass:: Org
+   :special-members: __init__
+   :members:
+   :inherited-members:
+.. autoclass:: OrgSet
```

Comment:
If users are confined to working within their current org, what is OrgSet for them? Is this for future functionality?

Reply:
yeah, this could be used for users who are members of multiple organizations. we're planning to add the ability to switch your active org via the SDK (instead of requiring a button-click in the browser)